chaco/color_mapper.py

```diff
@@ -304,7 +305,7 @@ def _get_color_bands(self):
         if self.color_depth is 'rgba':
             luts.append(self._alpha_lut)
 
-        result = zip(*luts)
+        result = list(zip(*luts))
```

Comment:
This also look wierd. It will also place a performance penalty on python 2.7./ Why not use `itertools.izip` in the first place